CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -144,31 +144,56 @@ void SiPixelLorentzAnglePCLHarvester::dqmEndJob(DQMStore::IBooker& iBooker, DQMS
   double min_drift_ = hists.h_drift_depth_adc_[1]->getAxisMin(1);
   double max_drift_ = hists.h_drift_depth_adc_[1]->getAxisMax(1);
 
-  iBooker.setCurrentFolder("AlCaReco/SiPixelLorentzAngleHarvesting/");
+  iBooker.setCurrentFolder(dqmDir_);
```

Comment:
This effectively changes the output directory, can you change the default in the `fillDescriptions` to be `AlCaReco/SiPixelLorentzAngleHarvesting`?

Reply:
I thought it should be consistent with that in the `SiPixelLorentzAnglePCLWorker.cc`, which is `AlCaReco/SiPixelLorentzAngle`, no?

Reply:
Not if we want to differentiate the worker and harvester histograms

Reply:
OK, I could change it back.